modules/local/kraken2/main.nf

```diff
@@ -0,0 +1,43 @@
+process KRAKEN2 {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
Could you change this to `process_high` to make sure we give enough resources for the larger databases we will be running for this pipeline?

Reply:
Sure can - completed in [f162cfe](https://github.com/phac-nml/speciesabundance/pull/18/commits/f162cfe3f9ac87bc6b02721ef06b70f5461fcde0) 😄

Reply:
Awesome, thanks so much 😄